docs/historias-de-usuario.md

```diff
@@ -0,0 +1,7 @@
+# Historias de Usuario
```

Comment:
¿Por qué ponéis esto? Incluid en el PR sólo lo que haga falta para evaluar.

Reply:
Si no lo has resuelto, no lo cierres. La forma correcta de hacerlo habría sido sacar esto del PR, ponerlo en un PR aparte y fusionarlo.